pkg/apis/extensions/v1beta1/types.go

```diff
@@ -342,6 +342,9 @@ type DeploymentStatus struct {
 
 	// Total number of unavailable pods targeted by this deployment.
 	UnavailableReplicas int32 `json:"unavailableReplicas,omitempty"`
+
+	// Total number of pods a deployment should have at any given point.
+	EnactedReplicas int32 `json:"enactedReplicas,omitempty"`
```

Comment:
Switched to nginx and suddenly this is not an issue. Nevermind. Going to debug the rest of the e2e test further tomorrow.